packages/react-server-cli/src/compileClient.js

```diff
@@ -13,93 +13,6 @@ import callerDependency from "./callerDependency"
 // import {logging} from "react-server"
 // const logger = logging.getLogger(__LOGGER__);
 
-// compiles the routes file for browser clients using webpack.
-// returns a tuple of { compiler, serverRoutes }. compiler is a webpack compiler
-// that is ready to have run called, and serverRoutes is a promise that resolve to
-// a path to the transpiled server routes file path. The promise only resolves
-// once the compiler has been run. The file path returned from the promise
-// can be required and passed in to reactServer.middleware().
-// TODO: add options for sourcemaps.
-export default (opts = {}) => {
```

Comment:
Ugh.  This is exactly what you said would happen, but it's just as painful as expected.

It's nice to have default exports at the top of modules!

I guess this is the price we pay for the safety this rule provides. 😖